Formula/crystalline.rb

```diff
@@ -0,0 +1,19 @@
+require "digest"
+
+class Crystalline < Formula
+  desc "A Language Server Protocol implementation for Crystal. "
+  homepage "https://github.com/elbywan/crystalline"
+  url "https://github.com/elbywan/crystalline/archive/v0.5.0.tar.gz"
+  sha256 "b7a203d0e5d4e37bbe744b371fca77868aca8f71"
+
+  depends_on "crystal"
+
+  def install
+    # TODO: How to specify LLVM_CONFIG value via depends on or at install time
+    system "make", "install", "PREFIX=#{prefix}"
```

Comment:
Would be great to hear ideas from other folks potentially about how to go about doing that. 

Running
```
LLVM_CONFIG=... make install
```

Works perfectly fine, but I don't think requiring variables is standard for homebrew installs.